app/assets/stylesheets/components/home_page_header.scss

```diff
@@ -1,25 +1,24 @@
-header {
-    padding-bottom: 60px;
+.hero {
+  text-align: center;
+  width: 100%;
 
-  h1 {
-    margin-bottom: 30px;
+  header {
+    padding-bottom: 3.75rem;
```

Comment:
I was pairing with one of the front enders at work, he was saying its best practice to use em's or rems for most dimensions instead of fixed pixel values. Apparently if we use em's for font size we should use it for margin and paddings as well. It helps keep things proportional with little extra work in responsive designs.

I've done a bit more research on it and came across this, apparently is best practice across the industry. I always thought it should be pixel values for anything related to box model as well.

You raise a good point about the user changing their font size 🤔

https://www.sitepoint.com/community/t/should-i-define-css-margins-in-pixels-or-ems/99716